README.md

```diff
@@ -1,2 +1,49 @@
-# tracker
-OpenFisca Web API usage tracker
+# OpenFisca-Tracker
+
+OpenFisca is a versatile microsimulation free software. 
+Check the online [documentation](https://doc.openfisca.fr) for more details.
+
+This package contains the tracker module of the [OpenFisca Web API](https://doc.openfisca.fr/openfisca-web-api/index.html). 
+It is an optional solution to track the calls on an installed OpenFisca Web API. It allows for usage analytics on a [Piwik](https://piwik.org) analytics plateform.
+
+## Ethics
```

Comment:
We need to move on on this issue, so I'm merging.
Feel free to edit this later if it needs improvement.